### PR TITLE
Cover gaps for zero-dim accessor

### PR DESCRIPTION
**Pull request by aleksmesh**
Covered gaps for zero-dim accessor.

### CODE REVIEW

File: tests/accessor/accessor_implicit_conversions.h
Comment (keryell) [inline tests/accessor/accessor_implicit_conversions.h:172]: Unused?

Comment (aleksmesh) [reply tests/accessor/accessor_implicit_conversions.h]: Fixed. Thanks.

**Comment by bader** [comment]
> Merged PR [#669](https://github.com/KhronosGroup/SYCL-CTS/pull/669) to fix CI and reduce accessor build and run time.

How is this supposed to be reviewed now? Is there an easy way to filter out changes from #669?

**Comment by keryell** [comment]
> How is this supposed to be reviewed now? Is there an easy way to filter out changes from #669?

I will wait for the other PR to land first.

**Comment by aleksmesh** [comment]
> > Merged PR [#669](https://github.com/KhronosGroup/SYCL-CTS/pull/669) to fix CI and reduce accessor build and run time.
> 
> How is this supposed to be reviewed now? Is there an easy way to filter out changes from #669?

I apologize for the troubles.
To review changes follow this link [908e8b](https://github.com/KhronosGroup/SYCL-CTS/pull/654/files/908e8b8f98f8b85e52ff847737dc7c0a3862e09c).

**Comment by bader** [comment]
#669 is merged, so I updated the branch and full pull request diff should work for code review.

**Comment by bader** [comment]
@aleksmesh, could you resolve merge conflicts, please?

**Review by ProGTX — APPROVED** [review]

**Comment by bader** [comment]
@aleksmesh, could you resolve new merge conflicts, please?
@keryell, please, approve if you have no objections to merge.

**Comment by aleksmesh** [comment]
> @aleksmesh, could you resolve new merge conflicts, please?

Merge conflict is resolved. Thanks.

**Review by keryell — APPROVED** [review]
Thanks.

**Comment by bader** [comment]
@aleksmesh, sorry, but we have new conflicts now. Could you take a look, please?

**Comment by aleksmesh** [comment]
@bader Conflicts are resolved.

**Pull request merged**
